packages/vscode-extension/src/treeviews/connectionTreeDataProvider.ts

```diff
@@ -155,16 +157,103 @@ export class ConnectionItem extends TreeItem {
     super(label, collapsibleState);
     this.tooltip = label;
     this.contextValue = this.type;
-
+    this.command = {
+      title: 'onClickConnect',
+      command: CONSTANTS.COMMANDS.CONNECT_COMMAND,
+      arguments: [this],
+    };
     switch (type) {
       case 'activeConnection':
+        this.iconPath = {
+          light: Uri.file(
+            path.join(
+              __dirname,
+              '..',
+              'resources',
+              'images',
+              'Neo4jActiveLight.svg',
+            ),
```

Comment:
Have you tried if the icons are visible in Windows?

You can generate a `.vsix` package doing:

```
cd packages/vscode-extension
npx vsce package
```

and then installing that VSIX manually when you click on the Marketplace icon in VSCode:

<img width="695" src="https://github.com/user-attachments/assets/f368f99b-4c0a-4a52-be27-0a57b3504d93" />